README.md

```diff
@@ -79,4 +79,4 @@ jobs:
 - You must provide a tag either via the action input or the git ref (i.e push / create a tag). If you do not provide a tag the action will fail.
 - If the tag of the release you are creating does not yet exist, you should set both the `tag` and `commit` action inputs. `commit` can point to a commit hash or a branch name (ex - `main`).
 - In the example above only required permissions for the action specified (which is `contents: write`). If you add other actions to the same workflow you should expand `permissions` block accordingly.
-- More info about why the default discussion category "Announcements" does not work with this action may be found here: https://github.com/goreleaser/goreleaser/issues/2304.
\ No newline at end of file
+- More info about why the default discussion category "Announcements" does not work with this action may be found here: https://github.com/goreleaser/goreleaser/issues/2304.
```

Comment:
Do you know what happened here? I don't see an obvious change. Maybe difference in line ending settings?